allennlp/data/dataset_readers/sequence_tagging.py

```diff
@@ -2,7 +2,7 @@
 
 from overrides import overrides
 
-from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
```

Comment:
as I understand it, it's because this module is not imported by `common/__init__.py` so you don't get the circular reference problem